.github/workflows/check-i18n.yml

```diff
@@ -21,6 +21,8 @@ jobs:
 
       - name: Set up Ruby environment
         uses: ./.github/actions/setup-ruby
+        with:
```

Comment:
If `libvips42` needs to be installed for every workflow, then it is best to add it to `.github/actions/setup-ruby/action.yml` and not use `additional-system-dependencies`.

The alternative is only requiring vips on the Ruby side in files where it is used, but I think I prefer everything to fail if vips is not installed, so admins will know if their setup is wrong.